incubator/hnc/cmd/manager/main.go

```diff
@@ -25,14 +25,15 @@ import (
 	"github.com/go-logr/zapr"
 	corev1 "k8s.io/api/core/v1"
 
-	// Change to use v1 when we only need to support 1.17 and higher kubernetes versions.
 	stdzap "go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
 	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	clientgoscheme "k8s.io/client-go/kubernetes/scheme"
 	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
 	ctrl "sigs.k8s.io/controller-runtime"
+	// Change to use v1 when we only need to support 1.17 and higher kubernetes versions.
```

Comment:
why do we change this comment here?

Reply:
auto format when package imported. I'll make it back